packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,144 @@
+  })
+
+  it('should create a transaction with Forward selection mode', async () => {
+    const options: SendTokensOptions = {
+      selectionMode: SelectionModeType.FORWARD
+    }
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({}, { [address]: ['2@DFI'] }, options)
+    await container.generate(1)
+
+    const account = await client.account.getAccount(address)
+
+    expect(account).toStrictEqual(['2.00000000@DFI'])
+    expect(typeof transactionHex).toStrictEqual('string')
+  })
+
+  it('should create a transaction with Crumbs selection mode', async () => {
+    const options: SendTokensOptions = {
+      selectionMode: SelectionModeType.CRUMBS
+    }
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({}, { [address]: ['2@DFI'] }, options)
+    await container.generate(1)
+
+    const account = await client.account.getAccount(address)
+
+    expect(account).toStrictEqual(['2.00000000@DFI'])
+    expect(typeof transactionHex).toStrictEqual('string')
+  })
+
+  it('should create a transaction with multiple destination address tokens', async () => {
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({}, { [address]: ['2@ETH', '0.1@DBTC', '10@DFI'] })
+    await container.generate(1)
+    const account = await client.account.getAccount(address)
+
+    expect(account).toStrictEqual(['10.00000000@DFI', '0.10000000@DBTC', '2.00000000@ETH'])
+    expect(typeof transactionHex).toStrictEqual('string')
+  })
+
+  it('should create a transaction with source address provided', async () => {
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({ [addA]: ['10@ETH'] }, { [address]: ['10@ETH'] })
+    await container.generate(1)
+
+    const account = await client.account.getAccount(address)
+
+    expect(account).toStrictEqual(['10.00000000@ETH'])
+    expect(typeof transactionHex).toStrictEqual('string')
+  })
+
+  it('should create a transaction with multiple source address tokens provided', async () => {
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({ [addA]: ['2@DBTC', '10@ETH'] }, { [address]: ['2@DBTC', '10@ETH'] })
+    await container.generate(1)
+
+    const account = await client.account.getAccount(address)
+
+    expect(account).toStrictEqual(['2.00000000@DBTC', '10.00000000@ETH'])
+    expect(typeof transactionHex).toStrictEqual('string')
+  })
+
+  it('should fail and throw an exception if destination address param is empty', async () => {
+    await expect(client.account.sendTokensToAddress({}, {})).rejects.toThrow('zero amounts in "to" param')
+  })
+
+  it('should throw an error with insufficient fund', async () => {
```

Comment:
If I run this test independently, it fails.

Reply:
Every `it` should pass if you run it independently. You probably need to change your code so the result is not depends on other tests

Reply:
Thanks for solving this. I have run all your test separately and all the tests are able to run independently.